.github/workflows/CI.yml

```diff
@@ -132,6 +136,11 @@ jobs:
           ccache --max-size=2G --set-config cache_dir=~/.ccache
           ccache --show-stats --set-config cache_dir=~/.ccache
 
+      - name: Install cmake
+        uses: aminya/install-cmake@master
```

Comment:
Nit:

```suggestion
        uses: aminya/install-cmake@6c511c28347701408b6b6c41773bea4280f90baa # v0.41.0
```

Reply:
https://github.com/aminya/setup-cpp is the authors maintained solution for this